cmd/scrape/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/google/go-github/github"
+	fixtures "github.com/src-d/lookout-test-fixtures"
+)
+
+// Scrape PRs for repo github.com/src-d/lookout-test-fixtures/
+const (
+	owner = "src-d"
+	repo  = "lookout-test-fixtures"
+)
+
+func main() {
+	tr := github.BasicAuthTransport{
+		Username: os.Getenv("GITHUB_USER"),
+		Password: os.Getenv("GITHUB_TOKEN"),
+	}
+
+	client := github.NewClient(tr.Client())
+
+	fmt.Println("Saving JSON fixtures...\n")
+
+	prs, _, err := client.PullRequests.List(context.Background(),
```

Comment:
I would really love to have a command to collect only 1 PR. It will be very useful for creating incremental fixtures.